server/service/src/sync/central_data_synchroniser.rs

```diff
@@ -35,6 +37,15 @@ impl CentralDataSynchroniser {
 
         let cursor_controller = CursorController::new(KeyType::CentralSyncPullCursor);
 
+        let msupply_central_server_id = KeyValueStoreRepository::new(connection)
+            .get_i32(KeyType::SettingsSyncCentralServerSiteId)?;
+
+        log::info!(
```

Comment:
Not strictly needed, maybe should make this a debug, but thought it was a helpful message to have in the logs?